api/controllers/FileUploadController.js

```diff
@@ -93,6 +93,17 @@ const FileUploadController = {
         }
     },
 
+    _maxFileLimitCheck(req) {
+        const totalFilesUploaded = req.session.eApp.uploadedFileData.length;
```

Comment:
Seems like you should be reusing HelperService.maxFileLimitExceeded here

Reply:
This is a far point. The reason this function does a similar thing is to trigger the error message to the user but I can incorporate the ` HelperService.maxFileLimitExceeded` function into this one.